src/plugins/chart_expressions/expression_xy/public/helpers/axes_configuration.ts

```diff
@@ -270,17 +269,3 @@ export function getAxesConfiguration(
 
   return axisGroups;
 }
-
-export function validateExtent(hasBarOrArea: boolean, extent?: AxisExtentConfig) {
-  const inclusiveZeroError =
-    extent &&
-    hasBarOrArea &&
-    ((extent.lowerBound !== undefined && extent.lowerBound > 0) ||
-      (extent.upperBound !== undefined && extent.upperBound) < 0);
-  const boundaryError =
-    extent &&
-    extent.lowerBound !== undefined &&
-    extent.upperBound !== undefined &&
-    extent.upperBound <= extent.lowerBound;
-  return { inclusiveZeroError, boundaryError };
-}
```

Comment:
Moved to `src/plugins/chart_expressions/expression_xy/public/helpers/validate_extent.ts`